config/auth_config.yml

```diff
@@ -0,0 +1,28 @@
+server:
+  addr: ':5001'
+  # TODO: not sure what it's for, the doc also have away for letsencrypt
+  certificate: '/path/to/server.pem'
+  key: '/path/to/server.key'
+
+token:
+  issuer: 'Docker auth issuer'
+  expiration: 900
+
+users:
+  # Password is specified as a BCrypt hash. Use htpasswd to generate.
+  'admin':
+    password: '' # Need generate a password
+  'ci':
+    password: '' # Need generate a password
+  '': {}
+
+acl:
+  - match: { account: 'admin' }
+    actions: ['*']
+    comment: 'Admin has full access to everything.'
+  - match: { account: 'ci' }
+    actions: ['push', 'pull']
```

Comment:
Will this be enough to also create new repos?

Reply:
From the [reference](https://github.com/cesanta/docker_auth/blob/main/examples/reference.yml), there is only `['pull', 'push', '*']` I guess 'push' will have right to create repo.